components/messenger.rst

```diff
@@ -136,6 +136,11 @@ through the transport layer, use the ``SerializerStamp`` stamp::
         ]))
     );
 
+.. tip::
```

Comment:
I would prefer a note directive here and get rid of the “Notice” of the sentence.

What do you think?

Reply:
Moved to a `note`, what do you think about the new wording?